packages/lesswrong/components/tagging/TagEditsByUser.tsx

```diff
@@ -37,14 +37,18 @@ const TagEditsByUser = ({userId, limit, classes}: {
     return <Components.Loading />
   }
 
-  if (results.length === 0) {
+  const resultsWithLiveTags = results
+    .filter(tagUpdates => tagUpdates.tag && !tagUpdates.tag.deleted)
```

Comment:
Went with frontend filtering as backend filtering would have involved changing the view into a custom resolver.